android/core/src/main/kotlin/com/chicagoroboto/features/sessions/SessionListPresenter.kt

```diff
@@ -42,4 +47,11 @@ class SessionListPresenter(private val sessionProvider: SessionProvider,
             favoriteProvider.removeFavoriteListener(it)
         }
     }
+
+    private fun findCurrentSessionIndex(sessions: List<Session>): Int {
+        // find the current session index
+        val now = Date()
+        val index = sessions.indexOfFirst { now.after(it.startTime) && now.before(it.endTime) }
```

Comment:
Random aside, as someone who still only dabbles in Kotlin, I was surprised that I couldn't find an alternate version of this method that returned index-or-null.